src/sentry/scripts/utils/sadd_capped.lua

```diff
@@ -7,6 +7,7 @@ local value = ARGV[1]
 local max_size = tonumber(ARGV[2])
 local ttl = ARGV[3]
 
+local existed = redis.call("EXISTS", key)
```

Comment:
It doesn't make any difference no, it's just a nit. But following up with the nit-picking here, we want to check if it didn't exist (instead of if it existed) -- if it exists the script does nothing.

Reply:
I see. I think the confusion came from me assuming the `EXISTS` wasn't the entry, but a specific value in the entry. Looks good to me then!